unlock-js/src/unlockProvider.js

```diff
@@ -105,4 +118,14 @@ export default class UnlockProvider extends providers.JsonRpcProvider {
     const data = UnlockPurchaseRequest.build(purchaseRequest)
     return this.signData(data)
   }
+
+  generateSignedEjectionRequest() {
```

Comment:
This method is the only real change in here -- the rest will disappear after rebasing

Reply:
easy enough!